Gemfile.lock

```diff
@@ -60,20 +60,26 @@ GEM
       coffee-script-source
       execjs
     coffee-script-source (1.12.2)
-    concurrent-ruby (1.1.5)
-    crass (1.0.5)
+    concurrent-ruby (1.1.6)
```

Comment:
Multiple areas where there are updated gemfiles

Reply:
Thanks for taking the time to set this up for us.  I appreciate it, and I'm sure everyone else does too!